engine/class_modules/sc_warrior.cpp

```diff
@@ -7092,35 +7092,35 @@ void warrior_t::apl_prot()
 
   default_list -> add_action( "auto_attack" );
   default_list -> add_action( this, "Charge", "if=time=0" );
-  default_list -> add_action( "heroic_charge", "if=runeforge.reprisal" );
+  default_list -> add_action( this, "heroic_charge,if=rage<=40");
```

Comment:
``
```suggestion
  default_list -> add_action( "heroic_charge,if=rage<=40" );
```

or 

```suggestion
  default_list -> add_action( this, "Heroic Charge", "if=rage<=40" );
```

current line is causing it to error out looking for an ability called "heroic_charge,if=rage<=40"